Dockerfile

```diff
@@ -22,13 +22,7 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-RUN groupadd -r user \
-    && adduser --disabled-login --system --shell /bin/false --uid 1000 --ingroup user user
-
-COPY --from=builder --chown=user:user \
-    /build/target/release/electrs .
-
-USER user
```

Comment:
There was a problem when the user running `electrs` was `user` - `electrs` couldn't open a database in `/mnt/electrs/config` due to insufficient privileges. 
I decided that the simplest solution would be not to specify the user so that `electrs` is run as `root`. Also we do the same in case of `relay`.